### ISSUE TITLE
How to re-check a 424 tag which is already placed on the reader?

### ISSUE THREAD

**Issue by FerGT50** [Obsttube/MFRC522_NTAG424DNA#2]
Hello,
I am using an ESP32 connected to a RFID-RC522 blue dev board via SPI. Arduino IDE with latest Arduino-ESP32 core.

Using your "Plain_ReadData" example, I am able to read the NDEF response from a 424 DNA tag, but only once: only the first time I place it on the reader.
After that, leaving the tag on the reader, I can not read it anymore.
I tried commenting out
```
else if (!ntag.PICC_IsNewCardPresent()){
      return;
```
but then I get an error from
`PICC_ReadCardSerial()` 

Long story short: is there a way to (periodically) keep querying the same tag I already have on the reader, without having to take it off and place it on the reader every time?

Many thanks

**Comment by FerGT50** [comment]
Workaround: comment out the "return" after !ntag.PICC_IsNewCardPresent() and force a deselect using PICC_TryDeselectAndWakeupA.

Not elegant, maybe there's a better way?
